From d4efd3393752a114c7b9d18ac5e65097d3eaaa21 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Fri, 27 Feb 2009 19:49:00 -0800
Subject: Staging: line6: fix bus_id usage

bus_id is now gone in the linux-next tree, so replace it with dev_name()
so the code works properly.

Cc: Markus Grabner <grabner@icg.tugraz.at>
Cc: Mariusz Kozlowski <m.kozlowski@tuxland.pl>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/line6/audio.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/staging/line6/audio.c
+++ b/drivers/staging/line6/audio.c
@@ -36,7 +36,8 @@ int line6_init_audio(struct usb_line6 *l
 
 	strcpy(card->driver, DRIVER_NAME);
 	strcpy(card->shortname, "Line6-USB");
-	sprintf(card->longname, "Line6 %s at USB %s", line6->properties->name, line6->ifcdev->bus_id);  /* 80 chars - see asound.h */
+	sprintf(card->longname, "Line6 %s at USB %s", line6->properties->name,
+		dev_name(line6->ifcdev));  /* 80 chars - see asound.h */
 	return 0;
 }
 
